polus-rolling-ball-plugin/src/requirements.txt

```diff
@@ -0,0 +1 @@
+scikit-image==0.18.1
```

Comment:
Once you change your base container to `python:3.9`, you should add `bfio[zarr]==2.1.7` to the requirements file.

Reply:
done